features/environment.py

```diff
@@ -4,6 +4,7 @@
 import os
 from rule_creation_protocol import protocol
 import json
+import orjson
```

Comment:
Why 2 json libraries? Can we just settle on one? If we use `orjson` we need to make sure all the requirements.txt are updated.

Reply:
[Reversed this change already in a later commit](https://github.com/buildingSMART/ifc-gherkin-rules/commit/f1d9f9b534bae5531cb7e5c9f26f8cdc8040d242). Orjson is a bit faster as the output is already in bytes. But I don't think time is really an issue here and it indeed led to some more unnecessary work updating libraries.